src/components/wheel/canvas/CanvasCollectionGrid.jsx

```
@@ -649,18 +681,45 @@ export function CanvasCollectionGrid({
+     * Ask for a frame.
+     *
+     * Cheap and idempotent by design — the interaction handlers call it on every
+     * scroll event and every pointer move, and outside saver mode it is a single
+     * `if` that does nothing because the loop was never parked.
+     */
+    const requestDraw = useCallback(() => {
+        if (!parkedRef.current || !renderRef.current) return;
+        parkedRef.current = false;
+        animationRef.current = requestAnimationFrame(renderRef.current);
+    }, []);
+    requestDrawRef.current = requestDraw;
```

ESLint

[error] Error: This value cannot be modified

Modifying a value used previously in an effect function or as an effect dependency is not allowed. Consider moving the modification before calling useEffect().

  715 |         animationRef.current = requestAnimationFrame(renderRef.current);
  716 |     }, []);
> 717 |     requestDrawRef.current = requestDraw;
      |     ^^^^^^^^^^^^^^ value cannot be modified
  718 |
  719 |     // A pull lands, a filter changes, or the player leaves saver mode: all three
  720 |     // are reasons the board no longer matches what is on the canvas.

Hint: If this value is a Ref (value returned by `useRef()`), rename the variable to end in "Ref".

(react-hooks/immutability)